sda-commons-dependencies/build.gradle

```diff
@@ -15,6 +15,8 @@ ext {
   weldVersion = '3.1.4.Final'
   openTracingUtilVersion = '0.33.0'
   scalaVersion = '2.12.11'
+  kafkaVersion = '2.2.1'
```

Comment:
The latest version is 2.5.0. Do we want to update this later through dependabot?

Reply:
Let's discuss our strategy for updating dependencies like Kafka / MongoDB / Postgresql. Version 2.2.x is the current version that is supported by IBM's default plan. We decided to update to this version in the last architectural decision: https://sda-se.atlassian.net/wiki/spaces/platform/pages/849510462/Synchronize+Kafka+Versions

Reply:
Can we add a comment to the line to tell that this was intentionally selected and should not be changed in the future?

Reply:
Or add it to the dependabot ignorelist? Otherwise it's hard to remember that there was this message that we shouldn't update it